### ISSUE TITLE
Google fonts

### ISSUE THREAD

**Issue by MattGreyDesign** [ms-studio/karmilla#47]
Do you have any plan to submit this font to Google fonts? It would be great to be able to have use of it from there and it is possible under the OFL license Karla holds. Karmilla has the potential of staying more up to date and comprehensive than Karla, so feeding it back to the community would be beneficial.

**Comment by ms-studio** [comment]
Possibly I could do that... but it would make even more sense if @google / @davelab6 would commission someone to make Karla more complete. This has been done for many other Google Fonts already, as far as I can see by following this discussion list: https://groups.google.com/group/googlefonts-discuss

**Comment by davelab6** [comment]
Yeah, I would like to do that, it just hasn't been prioritised yet, but
someday it will be. I'll keep it in mind for 2019 :

**Comment by MattGreyDesign** [comment]
The big problem I had with the original Karla was the strange tail of the lowercase "q" which you've "fixed", that's the only reason I don't use Karla.

**Comment by ms-studio** [comment]
If the original Karla gets a refresh in 2019, the distinctive q could be handled as a "stylistic alternate".
